src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -64,6 +64,10 @@ class Serializer implements SerializerInterface, ContextAwareNormalizerInterface
     private $denormalizerCache = array();
     private $normalizerCache = array();
 
+    /**
+     * @param NormalizerInterface[]|DenormalizerInterface[] $normalizers
```

Comment:
The syntax we previously used for such arrays is `(NormalizerInterface|DenormalizerInterface)[]`, like in https://github.com/symfony/symfony/blob/04b2c2db4f64174b9aca74f3cb60c84e55172807/src/Symfony/Component/Security/Core/Authentication/Token/UsernamePasswordToken.php#L30

But as it's still not supported by IDEs AFAIK 👍 to keep as is.